terraform/deploy/app/modules.tf

```diff
@@ -2,7 +2,7 @@ module "emr_ami" {
   source = "../../modules/amis"
 
   providers = {
-    aws = aws.management
+    aws = aws.management-ami
```

Comment:
Why was this changed? What problem did this solve?

Reply:
@IanUtley This was changed during the merge - had been added more recently so thought it was required. Can change back?

Reply:
Hmm, I think this decides which account contains the ami, and looking at the provider, this is always management, possibly ok  to keep.